deploy/docker/cp-edge/sync-routes.py

```diff
@@ -273,16 +286,48 @@ def append_system_endpoints(tool_endpoints, run_details):
                         current_tool_endpoint = json.loads(tool_endpoints[0])
                         current_tool_endpoint["isDefault"] = "true"
                         tool_endpoints[0] = json.dumps(current_tool_endpoint)
-
                 # Append system endpoints to the existing list
                 for system_endpoint in system_endpoints_matched:
-                        tool_endpoint = { "nginx": { "port": system_endpoint["endpoint"] }, "isDefault": "false" }
+                        tool_endpoint = { "nginx": { "port": system_endpoint["endpoint"] }}
+                        system_endpoint_port = system_endpoint["endpoint"]
+                        system_endpoint_name = None
                         if "friendly_name" in system_endpoint:
                                 tool_endpoint["name"] = system_endpoint["friendly_name"]
+                                system_endpoint_name = system_endpoint["friendly_name"]
                         if "endpoint_num" in system_endpoint and system_endpoint["endpoint_num"]:
                                 tool_endpoint["endpoint_num"] = system_endpoint["endpoint_num"]
+                        modified_tool_endpoints, modified_count, isDefaultEndpoint = \
```

Comment:
According to this naming, it looks like `modified_count` = `len(modified_tool_endpoints)` but this is not true. Maybe we should make more understandable names?

Reply:
More convenient prefixes are used for those values